jsapp/js/account/security/email/emailSection.module.scss

```diff
@@ -34,3 +34,8 @@
   display: flex;
   gap: 10px;
 }
+
+.emailText {
```

Comment:
The text is black in the designs:
![image](https://github.com/user-attachments/assets/ec9c4426-8cd7-43dd-95c6-bb228f0ffeba)

Also, can we use a more descriptive class name? Perhaps "readOnlyEmailText" or something along those lines.

Reply:
Just a reminder about the class name change here.

Reply:
Missed this one, thanks for reminder